webapp/pages/RepositoryDetails.jsx

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import {ResourceNotFound} from '../errors';
 import AsyncPage from '../components/AsyncPage';
 import RepositoryContent from '../components/RepositoryContent';
 import RepositoryHeader from '../components/RepositoryHeader';
 
 export default class RepositoryDetails extends AsyncPage {
   static contextTypes = {
     ...AsyncPage.contextTypes,
+    store: PropTypes.object.isRequired,
     repoList: PropTypes.arrayOf(PropTypes.object).isRequired
   };
 
   static childContextTypes = {
     ...AsyncPage.childContextTypes,
     ...RepositoryDetails.contextTypes,
-    repo: PropTypes.object.isRequired
+    store: PropTypes.object.isRequired,
```

Comment:
Isn't this already in context via `<Provider>`?

Reply:
this was a hack to work around enzyme yelling

the problem is i can do like ``<Provider store={mockedStore}><ComponentImTesting /></Provider>``, but then I cant pass context to the Component (as enzyme seems to not propagate it correctly).

It also throws up everywhere yelling about any context thats passed thats not explicitly defined in childContextTypes (per the above)

Reply:
I think you need to define `contextTypes` and `childContextTypes` in the 2nd arg to `mount`. e.g. https://github.com/getsentry/sentry/blob/master/tests/js/setup.js#L44